scripts/main.js

```diff
@@ -41,7 +42,9 @@ if(getAppName){
 	document.querySelector('#name-app').innerHTML = getAppName;
 }
 
-
+var apps = [ ];
+localStorage.setItem('apps','JSON.stringify(apps)');
+JSONparse(apps);
```

Comment:
What is this coed supposed to do?

```js
localStorage.setItem('apps','JSON.stringify(apps)');
```

sets the `apps` key to the string `''JSON.stringify(apps)''`. If you want to stringify the `apps` array leave out the surrounding quotes: `localStorage.setItem('apps', JSON.stringify(apps));`